packages/babel-standalone/examples/scriptTag-module.htm

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <title>babel-standalone example - External script</title>
+</head>
+<body>
+  Using Babel <strong id="version"></strong>:
+  <pre id="output">Loading...</pre>
+
+  <script src="../babel.js"></script>
+  <script type="text/babel/module" data-presets='es2016'>
```

Comment:
If we want to do this, we should use a mimetype parameter like `text/babel; goal=module`

Reply:
I'd be happy with that. I used `indexOf` because I saw it used here https://github.com/babel/babel/pull/8410/files#diff-7a71ff75920619cbc0e67883df263ca8L188. Is there anywhere in the codebase (or otherwise) you can point me toward regard parsing something like the mimetype you suggest?

Reply:
Gotcha, yeah that at least does `type.split(";")` so it's not as generic. I'd vote for 
```
script.type.split(';').slice(1).some(meta => /^\s*type\s*=\s*module\s*$/.test(meta))
```
probably.

Reply:
Cool, thanks! I will give it a go 👍